react/components/BuyButton/index.js

```diff
@@ -64,14 +65,25 @@ export const BuyButton = ({
   const translateMessage = useCallback(id => intl.formatMessage({ id: id }), [
     intl,
   ])
+  const orderFormItems =
```

Comment:
just use ramda's path...

Reply:
Yeeeah... in this case should be better :)